scrapy/utils/curl.py

```diff
@@ -7,6 +8,13 @@
 from w3lib.http import basic_auth_header
 
 
+class DataAction(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        value = str(values).encode("utf-8").decode("utf-8")
+        value = value[1::] if re.match(r"^\$(.+)", value) else value
```

Comment:
This feels over complicated.

Wouldn’t something like this work?

```suggestion
        value = str(values)
        if value.startswith("$"):
            value = value[1:]
```

Reply:
Yes, I just checked and it works too. Thanks